xcode/minizip/minizip.xcodeproj/project.pbxproj

```diff
@@ -189,6 +189,7 @@
 		474017FB23F5B2CE00A93C81 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				CODE_SIGN_IDENTITY = "";
```

Comment:
Is it ok?

Reply:
Yes. Modules are signed separately, this was a fix suggested by xcode.